### DISCUSSION TITLE
Question: Update premium themes to tailwind?

### DISCUSSION THREAD

**Discussion by Tilgoron** [primefaces/community#507] [PrimeNG Templates] [answered]
Hey guys, in our company we purchased PrimeNg Morpheus and we love working with your library. I wanted to ask if there are any future plans to migrate the premium themes to tailwind (In our case Morpheus). 
Thank you very much! :)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've plans to make an update regarding Tailwind & PrimeNG, we haven't discussed a Tailwind update about templates. But nice idea though, thanks for the idea and feedback!

**Reply by janeznavteh100** [reply]
Hi, do you have an estimate of when the Tailwind update for PrimeNg will be released?
